client/js/components/procedure/StatementSegmentsList/StatementSegmentsList.vue

```diff
@@ -420,6 +420,20 @@ export default {
       'toggleSlidebarContent'
     ]),
 
+    /**
+     * This prevents the user from unintentionally deleting an unsaved text by synchronizing the local
+     * statement in StatementMeta.vue (which also emits the local statement when saving only metadata)
+     * with the statements from store. The editor automatically updates the state of statements in the
+     * store when registering an input. This only occurs when a statement has not been segmented already.
+     *
+     * @param {object} statement - The local statement of StatementMeta.vue.
+     */
+    canUpdateStatementAttributesFullText (statement) {
```

Comment:
Why did you choose this name for the method? I'd rather expect it to be something like `synchronizeFulltext`